snippets/focus-within.md

```diff
@@ -0,0 +1,51 @@
+### Focus Within
+
+The psuedo class `:focus-within` gives focus to a parent element like a `form` element when `input` elements receive focus.
```

Comment:
I'd reword this to "... applies styles to a parent element if any child element gets focused. For example, ..."

Reply:
I did and now I don't know how to submit a new PR lol.